client/src/hooks/helpers/useGetValuesToDisplay.ts

```diff
@@ -0,0 +1,78 @@
+import useCryptoValues from 'hooks/queries/useCryptoValues';
+import useSettingsStore from 'store/settings/store';
+import getNumberWithSpaces from 'utils/getNumberWithSpaces';
+import getValueCryptoToDisplay, {
+  GetValueCryptoToDisplayProps,
+} from 'utils/getValueCryptoToDisplay';
+import getValueFiatToDisplay, { GetValueFiatToDisplayProps } from 'utils/getValueFiatToDisplay';
+
+type GetValuesToDisplayProps = {
+  coinPricesServerDowntimeText?: GetValueFiatToDisplayProps['coinPricesServerDowntimeText'];
+  showLessThanZeroFiat?: boolean;
+  valueCrypto?: bigint;
+  valueString?: string;
+} & GetValueCryptoToDisplayProps;
+
+type GetValuesToDisplay = (props: GetValuesToDisplayProps) => {
+  primary: string;
+  secondary?: string;
+};
+
+export default function useGetValuesToDisplay(): GetValuesToDisplay {
```

Comment:
Have you considered unit-testing this hook?

Reply:
I considered but i decided to drop it. This hook returns unmodified data from utils that are already tested (main logic is covered).